Resources/Prototypes/_Impstation/Entities/Structures/Specific/apid.yml

```diff
@@ -0,0 +1,59 @@
+- type: entity
+  id: ApidCryopodBotanist
```

Comment:
i was thinking of these cryopods when i mentioned the parenting (i did totally gloss over the orders though, good catch)